AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/AppLovin/MRECs/Interface Builder/ALDemoInterfaceBuilderMrecViewController.h

```diff
@@ -0,0 +1,13 @@
+//
+//  ALDemoInterfaceBuilderMrecViewController.h
+//  AppLovin MAX Demo App - ObjC
+//
+//  Created by Nana Amoah on 11/22/21.
+//  Copyright © 2021 AppLovin Corporation. All rights reserved.
+//
+
+#import "ALBaseAdViewController.h"
+
+@interface ALDemoInterfaceBuilderMrecViewController : ALBaseAdViewController
```

Comment:
You should check in the relevant lines in the `.xcodeproj` file, otherwise we can't access these new files.

Reply:
Also, in code, our convention is `*MRec*` as opposed to `*Mrec*`.